spec/controllers/activities_controller_spec.rb

```diff
@@ -88,6 +88,16 @@
           expect(response).to have_http_status(:not_found)
         end
       end
+
+      describe "inactive user activity" do
+        let!(:inactive_user) { create(:user, status: User.statuses[:locked]) }
```

Comment:
That's valid.

You can also use `create(:locked_user) next time. It takes care of setting the status to `User.statuses[:locked]` among other things.